File: auto-docs/executables/python/overlaid_histogram.py
# Learn about API authentication here: plot.ly/python/getting-started
# Find your api_key here: plot.ly/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

import numpy as np
x0 = np.random.randn(500)
x1 = np.random.randn(500)+1

py.sign_in('TestBot', 'r1neazxo9w')
trace1 = Histogram(
    x=x0,
    opacity=0.75
)
trace2 = Histogram(
    x=x1,
    opacity=0.75
)
data = Data([trace1, trace2])
layout = Layout(
    barmode='overlay'
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, filename='overlaid-histogram', auto_open=False)

File: auto-docs/executables/nodejs/overlaid_histogram.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var x0 = [];
var x1 = [];

for (var i = 0; i < 500; i ++) {
	x0[i] = Math.random();
	x1[i] = Math.random() + 1;
}

var trace1 = {
  x: x0, 
  opacity: 0.75, 
  type: "histogram"
};
var trace2 = {
  x: x1, 
  opacity: 0.75, 
  type: "histogram"
};
var data = [trace1, trace2];
var layout = {barmode: "overlay"};

var graph_options = {filename: "overlaid-histogram", fileopt: "overwrite", layout: layout, auto_open: false}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});